Pod/Classes/VOKMappableModel.h

```diff
@@ -0,0 +1,32 @@
+//
+//  VOKMappableModel.h
+//  VOKCoreData
+//
+
+#import "VOKCoreDataManager.h"
+
+/**
+ *  Any models that conform to this protocol will be automatically registered for mapping with the shared instance 
```

Comment:
I was thinking about it more as: (1) the comment and behavior should be aligned; and (2) as it is (in `init`), it's forced on all instances, whereas if I moved it to the shared-instance's `dispatch_once`, it'd only be on the shared instance, so someone hellbent on not using it could use a different instance.

As to performance, setting up a mapper is mostly property-setting and no computation, except that when the array of maps is set and then again when the unique key is set, it loops over the array of maps to get the foreign key matching the unique key (and it looks like that loop isn't breaking when it finds it, so that can be optimized a bit), so I'd expect it all to run very quickly, even with lots of maps or large maps, but I'll see if I can measure it on-device sometime today.